aspnet/hosting/directory-structure.rst

```diff
@@ -0,0 +1,19 @@
+.. _directory-structure:
+
+Directory Structure
+===================
+
+By `Sourabh Shirhatti`
+
+
+In ASP.NET 5, the application directory comprises of three sub-directories. This is unlike previous versions of ASP.NET where the entire application lived inside the web root directory.
+
+=======   ===========
+Folder    Description     
+=======   ===========  
+approot   Contains the application, app config files, packages and the DNX runtime.
+logs      The default folder for HTTP PlatformHandler to redirect logs to.
```

Comment:
When you run `dnu publish`, it adds the `stdoutLogFile` property to `httpPlatform` attribute to point to the logs directory by default.